site/docs/BestPractice-ServingOverHTTP.md

```diff
@@ -44,7 +44,8 @@ A standard GraphQL POST request should use the `application/json` content type,
 ```js
 {
   "query": "...",
-  "variables": { variable1: value, ... }
+  "oprationName": "...",
```

Comment:
I think you're missing the `e` in `operationName` here and also above.

Also, I think it's worth mentioning that the `operationName` in the POST body is optional, and only required if you have multiple operations?

As an additional wrinkle, express-graphql doesn't support multiple operations in one document as far as I know.